37_environment_lab/envlab.txt

```diff
@@ -2,14 +2,26 @@
   - ll => ls -l
   - rm => rm -i
   - cp => cp -i
-
+ll="ls -i"
+rm="rm -i"
+cp="cp -i"
+  
 2. How can you "install" all defined aliases into the current shell? How can you add them permenantly to all shells?
+In the current shell: just add alias as needed, such as: alias ll="ls -a". 
+Also can use a bash file to add several aliases and then run the file
+
+For all shells, there is a need to edit the file .bashrc that is under the home directory
```

Comment:
just a quick note: ```.bashrc``` is used by bash, not all shells.. 
```.profile``` is used by multiple shells.